cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -166,10 +189,21 @@ ensure_caBundle "mutatingwebhookconfiguration"   "network-resources-injector-con
 envsubst < $MANIFESTS_DIR/network_config_policy.yaml | _kubectl create -f -
 
 sriov_operator_namespace="sriov-network-operator"
```

Comment:
These are not configurable nor dynamically created. Would you please use UPPER_CASE to signalize those are constants?

Reply:
np